docs/root/version_history/version_history.rst

```diff
@@ -7,16 +7,21 @@ Version history
   :titlesonly:
 
   current
+  v1.16.2
```

Comment:
Side question - do your instructions include  updating the website?  I remember adding them to the security release instructions but not sure if they're an obvious step from the stable release instructions. please let me know if they're not :-)

Reply:
Are you referring to security advisories: https://github.com/envoyproxy/envoy/security/advisories page?

Reply:
Nope, there's even more process :-)  

Last time I cut the quarterly release we realized we hadn't been updating the website for point releases.  Checking now that I looked it doesn't look like it was in https://docs.google.com/document/d/1uX3PcB1jE69Nxl4Tatm2sXdOzq3bIXUe426HcU82ILI/edit#

fixed!